components/ide/jetbrains/backend-plugin/BUILD.yaml

```diff
@@ -55,6 +55,7 @@ packages:
       - NO_VERIFY_JB_PLUGIN=${noVerifyJBPlugin}
     config:
       commands:
+        - ["mv", "build.gradle-stable.kts", "build.gradle.kts"]
```

Comment:
we can instead change build.sh to provide different build-file see https://discuss.gradle.org/t/multiple-build-files-for-one-project/31582